app/build.gradle.kts

```diff
@@ -11,14 +11,14 @@ android {
     buildToolsVersion = libs.versions.android.buildTools.get()
 
     defaultConfig {
-        applicationId = "org.tiqr.authenticator"
-        versionCode = 30
-        versionName = "4.0.0"
+        applicationId = "org.nl.sample"
```

Comment:
maybe org.tiqr.core or something like that?

Reply:
Doesn't this effect the notification flow?

Reply:
@sarahachem maybe net.surf.core ? or nl.serfnet.core. ask customer plz

Reply:
No, it's still an organisation. So domain should be org and surfnet isn't the name anymore, it's surf.

Reply:
correction, url is https://www.surf.nl/ so name should be nl.surf -> nl.surf.sample